### PR TITLE
chore: add openshift-checker 0.1.1 release

### PR DESCRIPTION
**Pull request by feloy**
Add the openshift-checker 0.1.1 release to the catalog

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: api/extensions/redhat/openshift-checker/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Docker OpenShift Analyzer
```
Comment (jeffmaury) [inline api/extensions/redhat/openshift-checker/0.1.0/README.md:1]: Should be updated ?

Comment (feloy) [reply api/extensions/redhat/openshift-checker/0.1.0/README.md:1]: I suppose we need to create a new release 0.1.1 to update the readme?

Comment (benoitf) [reply api/extensions/redhat/openshift-checker/0.1.0/README.md:1]: I think a new version is needed anyway for the license as it's not really clear on the repository as it's mixing ASL 2 and EPL2

**Review by benoitf — APPROVED** [review]

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**